cube/TaskDoneEvent.js

```diff
@@ -0,0 +1,49 @@
+cube(`TaskDoneEvent`, {
+  sql: `SELECT * FROM public.task_event WHERE name='task:done'`,
```

Comment:
It's out of the scope of this pull request, but maybe having this info about when the task was completed in the `task_event` table makes things a little bit complicated. Maybe we could also have a `task.completed_at` column?